cfgov/jinja2/v1/_includes/organisms/job-listing-list.html

```diff
@@ -25,22 +25,24 @@
     <h3>
         {{ heading or 'Career Openings' }}
     </h3>
-    <ul class="m-list m-list__unstyled">
+    <ul class="m-list m-list__unstyled m-list__links">
         {% for job in careers %}
         <li class="m-list_item">
             <a class="m-list_link"
-               href="{{ get_protected_url(job) }}">{{ job.title }}</a>
-            <p class="a-date">
-                CLOSING
-                {% import 'macros/time.html' as time %}
-                {{ time.render(job.close_date, {'date': true}) }}
-            </p>
+               href="{{ get_protected_url(job) }}">{{ job.title }}
```

Comment:
Trivial: It would be preferable to drop `{{ job.title }}` to its own line here, so it's not glossed over and mistaken for part of the `href`.